### PR TITLE
Update the Readme and add a schedule

### PR DESCRIPTION
**Pull request by aguspesce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,94 @@
...
+| 1°   | Installation       | Check the python installation.                                                    |
+| 2°   | The shell          | What is a command shell and why would I use one? Files, directories and locations |
+| 3°   | First part of Git  | What is? Set up. Hoe to use it?                                                   |
+| 4°   | Python basics      | What is ? Variables, types data, structure, Numpy                                 |
+| 5°   | Matplotlib         | ???                                                                               |
+| 6°   | Functions and more | For loops, conditional statements, factions                                       |
+| 7°   | Pandas             | ???                                                                               |
+| 8°   | Second part of Git | ???                                                                               |
+| 9°   | Rasterio/GDAL      | ???                                                                               |
+| 0°   | rioxarray/xarray   | ???                                                                               |
```
[the remark this answers is not in the document]
Comment (aguspesce) [reply README.md:24]: @mdpiper, It is work in progress... I forgot to put WIP in the title. We need to talk about this in the next meeting

**Comment by mdpiper** [comment]
We should remove the section [Installing Python Using Anaconda | Instalación de Python con Anaconda](https://github.com/CodeToCommunicate/CoCoLessons/tree/readme#installing-python-using-anaconda--instalaci%C3%B3n-de-python-con-anaconda) cuz we'll cover this in the new [setup lesson](https://github.com/CodeToCommunicate/CoCoLessons/blob/mdpiper/setup-local-coding-environment/coco_0_coding_environment.md#python-distribution).

**Comment by aguspesce** [comment]
@mdpiper 

>  We should remove the section [Installing Python Using Anaconda | Instalación de Python con Anaconda](https://github.com/CodeToCommunicate/CoCoLessons/tree/readme#installing-python-using-anaconda--instalaci%C3%B3n-de-python-con-anaconda) cuz we'll cover this in the new [setup lesson](https://github.com/CodeToCommunicate/CoCoLessons/blob/mdpiper/setup-local-coding-environment/coco_0_coding_environment.md#python-distribution).

I will change this. So, in the README I will put a reference to the setup file that you created.

**Comment by mdpiper** [comment]
> I will change this. So, in the README I will put a reference to the setup file that you created.

@aguspesce Yes, please--thank you!

**Pull request merged**